### PR TITLE
ADEN-10878 Fixing UCX Desktop issues

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: platforms/ucp-desktop/styles.scss [hunk 1]
```diff
@@ -20,6 +20,8 @@
 		#hivi_leaderboard {
 			@include bfaa-slot();
 			@include sticky-tlb-slot();
+
+			z-index: 1000;
```
Comment (nandy-andy) [inline platforms/ucp-desktop/styles.scss:24]: Can't we use a variable here? 🙏

File: src/ad-engine/models/ad-slot.ts [hunk 2]
```diff
@@ -44,6 +44,7 @@ export interface SlotConfig {
 	insertBeforeSelector?: string;
 	insertAfterSelector?: string;
 	parentContainerSelector?: string;
+	parentContainerPosition?: insertMethodType;
```
Comment (Loczi94) [inline src/ad-engine/models/ad-slot.ts:47]:
```suggestion
	insertIntoParentContainerMethod?: insertMethodType;
```

File: platforms/ucp-desktop/setup/context/slots/ucp-desktop-slots-context.setup.ts [hunk 3]
```diff
@@ -71,7 +71,8 @@ export class UcpDesktopSlotsContextSetup implements DiProcess {
 				aboveTheFold: true,
 				slotNameSuffix: '',
 				group: 'MR',
-				parentContainerSelector: '#rail-boxad-wrapper',
+				parentContainerSelector: '.main-page-tag-rcs, #rail-boxad-wrapper',
+				parentContainerPosition: 'prepend',
```
Comment (Loczi94) [inline platforms/ucp-desktop/setup/context/slots/ucp-desktop-slots-context.setup.ts:75]:
```suggestion
				insertIntoParentContainerMethod: 'prepend',
```

File: src/ad-engine/services/slot-injector.ts [hunk 4]
```diff
@@ -40,7 +40,9 @@ class SlotInjector {
 		const slotConfig: SlotCreatorConfig = {
 			slotName,
 			anchorSelector: config.insertBeforeSelector || config.parentContainerSelector,
-			insertMethod: config.insertBeforeSelector ? 'before' : 'append',
+			insertMethod: config.insertBeforeSelector
+				? 'before'
+				: config.parentContainerPosition || 'append',
```
Comment (Loczi94) [inline src/ad-engine/services/slot-injector.ts:45]:
```suggestion
				: config.insertIntoParentContainerMethod || 'append',
```

File: platforms/ucp-desktop/styles.scss (hunk 1 above)
Comment (dmnsobczak) [reply platforms/ucp-desktop/styles.scss]: Yep, added :)

**Review by Loczi94 — APPROVED** [review]

**Pull request merged**